src/pymor/bindings/dunegdt.py

```diff
@@ -0,0 +1,189 @@
+from pymor.core.config import config
+
+
+if config.HAVE_DUNEGDT:
+
+    from collections import OrderedDict
+    import os
+    import subprocess
+    import tempfile
+    from tempfile import mkstemp
+
+    from dune.xt.common.vtk.plot import plot as k3d_plot
+    from dune.gdt import DiscreteFunction
+
+    from pymor.core.base import ImmutableObject
+    from pymor.bindings.dunext import DuneXTVector
+    from pymor.vectorarrays.list import ListVectorArray
+
+
+    class DuneGDTParaviewVisualizer(ImmutableObject):
+        """Visualize a dune-gdt discrete function using paraview.
+
+        Parameters
+        ----------
+        space
+            The dune-gdt space for which we want to visualize DOF vectors.
+        """
+
+        def __init__(self, space):
+            self.__auto_init(locals())
+
+        def visualize(self, U, m, title=None, legend=None, separate_colorbars=False,
+                      rescale_colorbars=False, block=None, filename=None, columns=2):
+
+            def visualize_single(vec, vec_name, f_name):
+                discrete_function = DiscreteFunction(self.space, vec, vec_name)
+                discrete_function.visualize(f_name[:-4])
+
+            suffix = '.vtp' if self.space.dimDomain == 1 else '.vtu'
+            interactive = filename is None
+            to_clean_up = []
+
+            if isinstance(U, tuple):
+                # we presume to have several vectors which should be visualized side by side
+                assert all([isinstance(u, ListVectorArray) for u in U])
+                assert all([len(u) == 1 for u in U])
+                assert all([all([isinstance(v, DuneXTVector) for v in u._list]) for u in U])
+                if legend is not None and len(legend) == len(U):
+                    names = legend
+                elif title is not None and len(title) == len(U):
+                    names = title
+                else:
+                    names = list()
+                    for ii in range(len(U)):
+                        names.append('STATE_{}'.format(ii))
+                data = OrderedDict()
+                if interactive:
+                    for ii in range(len(U)):
+                        _, filename = mkstemp(suffix='_{}{}'.format(ii, suffix))
+                        data[filename] = U[ii]._list[0]
+                else:
+                    if (filename.endswith('.vtp') or filename.endswith('.vtu')):
+                        filename = filename[:-4]
+                    assert len(filename) > 0
+                    for ii in range(len(U)):
+                        data['{}_{}{}'.format(filename, ii, suffix)] = U[ii]._list[0]
+                for name, f_name in zip(names, data):
+                    visualize_single(data[f_name].impl, name, f_name)
+                if interactive:
+                    _, pvd_filename = mkstemp(suffix='.pvd')
+                    with open(pvd_filename, 'w') as pvd_file:
+                        pvd_file.write('<?xml version="1.0"?>\n')
```

Comment:
This can be simplified using a `VtkGroup`, see vtkio.py:38

Reply:
Oh, nice. Would I simply call `groud.addFile`?

Reply:
Ah, but this relates to your next comment ...

Reply:
> Oh, nice. Would I simply call `groud.addFile`?

Right. Basically `create Group -> addFile, addFile,... -> save`

Reply:
Not gonna happen since you will update the visualizers anyway.